jina/docker/helper.py

```diff
@@ -63,5 +63,5 @@ def sleep(self, seconds=5):
         self.logger.debug(f'sleeping for {seconds} seconds')
         time.sleep(seconds)
 
-    def __exit__(self, type, value, traceback):
```

Comment:
this is inherited from the exit stack builtin, i don't think we need to change that

Reply:
by inherited, i mean if you type `__exit__` in Pycharm, it auto-complete with this signature

Reply:
okay my bad, somehow this one is not correctly auto-filled by Pycharm. The correct signature should be 
`object.__exit__(self, exc_type, exc_value, traceback)`

let's change to that
https://docs.python.org/3.7/reference/datamodel.html#object.__exit__

Reply:
thanks, I changed it